app/models/form526_job_status.rb

```diff
@@ -5,14 +5,17 @@ class Form526JobStatus < ApplicationRecord
 
   alias submission form526_submission
 
+  FAILURE_STATUSES = {
+    non_retryable_error: 'non_retryable_error',
+    exhausted: 'exhausted'
+  }.freeze
   STATUS = {
     try: 'try',
     success: 'success',
     retryable_error: 'retryable_error',
     non_retryable_error: 'non_retryable_error',
-    exhausted: 'exhausted',
     pdf_not_found: 'pdf_not_found'
-  }.freeze
+  }.merge(FAILURE_STATUSES).freeze
```

Comment:
Did you also intend to remove 'non_retryable_error' from STATUS as it's also now designated as one of the FAILURE_STATUSES?

Reply:
yes, to my mind `exhausted` and  `non_retryable_error` are job failure statuses. Is that incorrect?  I know that's what we've been using for our weekly reports for a long time

Reply:
Their both failures. I was just wondering if you wanted to delete L16 as `non_retryable_error` is included in FAILURE_STATUSES.

Reply:
ah derp, ok i get it. yes good call